component/libfuse/libfuse.go

```diff
@@ -332,6 +332,14 @@ func (lf *Libfuse) Configure(_ bool) error {
 		return fmt.Errorf("%s config error %s", lf.Name(), err.Error())
 	}
 
+	// Disable libfuse logs if the mount is not running in foreground.
+	// Currently as of 01-05-2025, we emit the libfuse logs only to the stdout.
+	if !common.ForegroundMount {
+		if lf.traceEnable {
```

Comment:
You shall never be using fuse trace enable unless there is a failure in init of the fuse itself. This is not a general purpose option that customers are suppose to use, its for very high level debugging to look at any change in fuse driver causing the mount failure.